README.md

```diff
@@ -1 +1,159 @@
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Backend</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby on rails</a></li>
+    <li><a href="https://www.ruby-lang.org/en/">Postgresql Database</a></li>
+  </ul>
+</details>
+<details>
+<summary>Frontend</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Html</a></li>
+    <li><a href="https://www.ruby-lang.org/en/">CSS</a></li>
+    <li><a href="https://www.ruby-lang.org/en/">Bootstrap</a></li>
+  </ul>
+
+</details>
+
+### Demo <a name="demo-link"></a>
+
+- Video demo
+https://www.loom.com/share/a90fab36e4ef434598cc7991f77b1b7b?sid=d098e8b0-0bbb-4c95-a99c-9e68bfd4fdd1
+
+- Project live on Server
+    https://budgetapp-htg3.onrender.com
+
+### Key Features <a name="key-features"></a>
+
+- Built MVC using Rails framework
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+## Prerequisites <a name="prerequisites">
+
+- Ruby installed on your system
+- Rails installed on your system
+- Postgresql installed on system
+
+## Installation <a name="install">
+
+1. Clone this repository to your local machine:
+
+   ```bash
+   git clone https://github.com/zbahati/budgetApp
+   ```
+
+2. Navigate to the project directory:
+
+   ```bash
+   cd budgetApp
+   ```
+3. Run this command to start application in browser on the port http:\\127.0.0.1:300:
+
+   ```bash
+   rails s
+   or
+   rails server
+
+   ```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Testing <a name="testing">
+
+1. To run the tests for this project:
+
+  ```bash
+  rspec
+  or
+  bundle exec rspec
+  ```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Bahati**
+
+- GitHub: [@githubhandle](https://github.com/zbahati/)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/zirimwabagabo-bahati/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+ - Add UI
+ - Make UI responsive to the different devices
+ - Add test
+ - seach using ransack
+ - cancan for authorization
+ - kaminari for pagination
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/zbahati/recipeApp/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, you can support me by clicking on the start
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for this learning and reviewing our progress
+
```

Comment:
- Hey, please kindly give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) to meet this missing part of the project requirement.